### PR TITLE
Full Text Search Query

### PR DESCRIPTION
**Pull request by dplewis**
Ref: https://github.com/parse-community/parse-server/pull/3904
Closes: https://github.com/parse-community/Parse-SDK-iOS-OSX/issues/1167

### CODE REVIEW

File: Parse/PFQuery.h [hunk 1]
```diff
@@ -197,6 +197,17 @@ typedef void (^PFQueryArrayResultBlock)(NSArray<PFGenericObject> *_Nullable obje
...
 /**
+ Add a constraint for finding string values that contain a provided
+ string using Full Text Search
+
+ @param key The key to be constrained.
+ @param value the substring that the value must contain.
+
+ @return The same instance of `PFQuery` as the receiver. This allows method chaining.
+ */
+- (instancetype)whereKey:(NSString *)key fullText:(NSString *)text;
```
Comment (flovilmart) [inline Parse/PFQuery.h]: Can we call it matchesText instead of fullText?

Comment (dplewis) [reply Parse/PFQuery.h]: done

**Comment by flovilmart** [comment]
Awesome! Let’s make a plan for the 3.0 release so we can release everything at once :)

**Comment by dplewis** [comment]
Works for me

**Comment by montymxb** [comment]
Looks like this is just about good, and the requested changes have been added some time ago now...I'll assign myself and give this a better look in a bit.

**Comment by johnantoni** [comment]
Hi @montymxb as the changes have been approved can we get this one merged in?

**Comment by montymxb** [comment]
@johnantoni Yes we can! Seems we didn't follow up after approval...

**Pull request merged**